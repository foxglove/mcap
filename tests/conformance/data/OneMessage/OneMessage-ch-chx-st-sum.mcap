version https://git-lfs.github.com/spec/v1
oid sha256:894450eb4187e8695e36e9aac400d6042f3c2d8d902cece801836bb8427d58f9
size 437
