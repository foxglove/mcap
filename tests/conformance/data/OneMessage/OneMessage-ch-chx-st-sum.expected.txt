Header library=s:"" profile=s:""
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Message channel_id=i:1 data=b:<010203> log_time=n:2 publish_time=n:1 sequence=i:10
Statistics attachment_count=i:0 channel_count=i:1 channel_message_counts=m:{i:1=n:1} chunk_count=i:1 message_count=n:1
ChunkIndex chunk_length=n:162 chunk_start_offset=n:25 compressed_size=n:115 compression=s:"" end_time=n:2 message_index_length=n:0 message_index_offsets=m:{} start_time=n:2 uncompressed_size=n:115
SummaryOffset group_length=n:0 group_opcode=i:13 group_start=n:245
SummaryOffset group_length=n:43 group_opcode=i:11 group_start=n:202
SummaryOffset group_length=n:77 group_opcode=i:8 group_start=n:245
Footer summary_crc=i:0 summary_offset_start=n:322 summary_start=n:202
